### ISSUE TITLE
Insufficient colour contrast in the example on the template for the component detail page

### ISSUE THREAD

**Issue by furstenberg** [sl-design-system/components#212]
WCAG Success Criterion 1.4.11 (Non-text Contrast) states that the visual presentation of user interface components have a contrast ratio of at least 3:1 against adjacent color(s).

I believe the current example with the button would fail when dark mode is enabled. The current contrast between the button and the background is less than 3:1.

![button-example](https://user-images.githubusercontent.com/18802/213992818-c176c7d3-c0fd-4ff4-8320-281fe8647e72.png)

Expected outcome would be a colour contrast of at least 3:1.

```[tasklist]
### Tasks
- [x] check color contrast in light mode
- [x] check color contrast in dark mode
- [x] Improve color tokens
- [x] Testing
```
